dc-cudami-server/dc-cudami-server-backend-api/src/main/java/de/digitalcollections/cudami/server/backend/api/repository/identifiable/entity/parts/SubtopicRepository.java

```diff
@@ -30,4 +30,18 @@
   Subtopic saveWithParentTopic(Subtopic subtopic, UUID parentTopicUuid);
 
   Subtopic saveWithParentSubtopic(Subtopic subtopic, UUID parentSubtopicUuid);
+
+  List<Subtopic> getSubtopicsOfEntity(UUID entityUuid);
```

Comment:
Fehlt hier nicht noch dieselbe Methode mit einer `Entity`?

Reply:
ja, hab ich ergänzt. und ausserdem alle default-methoden aus der impl in dieses interface verlagert